frontend/src/components/Dialogs/ConfirmDialog.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-dialog
+    class="confirm-dialog"
+    content-class="confirm-dialog"
+    :model-value="dialog"
+    :fullscreen="$vuetify.display.mobile"
+    @update:model-value="close">
+    <v-card height="100%" class="d-flex width-90">
```

Comment:
I think you can pass title and subtitle here as props so the template is cleaner and uses fewer lines. Same with text really.